test/podFailover/README.md

```diff
@@ -0,0 +1,55 @@
+## Pod Failover Test
+This test run creates a controller pod and a workload pod. The controller pod is responsible for repeatedly creating and deleting workload pod over the given duration. The workload pod will publish a metric called "default_failover_testing_workload_downtime" with the value of the downtime seen by the workload pod in seconds.
+
+## Prerequisite
+- Make sure a kubernetes cluster(with version >= 1.13) is set up and kubeconfig is under $HOME/.kube/config.
```

Comment:
AKS only supports 1.18 or later, so that should be the recommended minimum K8s version.